client/dashboard/components/summary-button-list/style.scss

```diff
@@ -41,5 +41,11 @@
 			border-bottom-left-radius: inherit;
 			border-bottom-right-radius: inherit;
 		}
+
+		// Ensure focused elements appear above adjacent hovered elements.
+		.dashboard-summary-button-list__children-list-item > *:focus {
```

Comment:
Should we also match `:focus-visible` for good measure?

Reply:
Done. Thanks!